### ISSUE TITLE
No memory of last session

### ISSUE THREAD

**Issue by Peter-Karimeddini** [ClimberAB/ClimberCustomReport#60]
### Description

Unlike v1, v2 does not remember the dimensions/measures chosen from your last session. I understand the new paid bookmark functionality provides even an even better version of this feature, but the removal of automatic recall of the previous session is significant. Even if a user works around this by duplicating the sheet to create their own Default State, they have to know they will want this before they start since duplicating the sheet does not carry over the chosen dimensions/measures (making them select the desired dimensions/measures twice). Even if the paid bookmark option was used, this requires the user to remember to use it whereas before it was automatic (albeit only for 1 set).

### Steps to Reproduce

1. Install Custom Report.
2. Add any data.
3. Load data.
3. Add a master table with some dimensions.
4. Add a Custom Report object.
5. Choose dimensions & measures.
6. Close the report.
7. Reopen the report.

##### Expected behavior

The dimensions & measures selected in the previous session should persist.

##### Actual behavior

The dimensions & measures are reset.

### Environment
```
[X] Desktop
[X] Enterprise
```
### Versions

* Extension version: 2.0.3 & 2.1.0
* Browser: Chrome 67
* Qlik Sense: Nov 2017
* Operating system: Win 10

**Comment by MichaelClimber** [comment]
Hi Peter, 
thank you for your input. 
Remembering the previous session caused a lot of questions and confusion from users so based on that feedback we decided it was better to remove it. As you mention, the bookmarks and presets available in the Plus version is a much better option since it is less confusing and possible to control. With the change we have also made some significant improvements in how the current session is handled (for example the column width is now saved within the session in the free version) and we have less errors. All in all we are happy with the change and it is unlikely to revert back to the way it was but we will of course consider if there is some other alternative solution for your use-case.
Kind regards
Michael
